[2023-10-29T22:17:53.563+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v3.first_task scheduled__2022-03-25T00:00:00+00:00 [queued]>
[2023-10-29T22:17:57.476+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v3.first_task scheduled__2022-03-25T00:00:00+00:00 [queued]>
[2023-10-29T22:17:57.655+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-10-29T22:18:02.025+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): first_task> on 2022-03-25 00:00:00+00:00
[2023-10-29T22:18:02.459+0000] {standard_task_runner.py:57} INFO - Started process 839 to run task
[2023-10-29T22:18:02.820+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'our_dag_with_python_operator_v3', 'first_task', 'scheduled__2022-03-25T00:00:00+00:00', '--job-id', '3488', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmp2vuqzp1o']
[2023-10-29T22:18:03.118+0000] {standard_task_runner.py:85} INFO - Job 3488: Subtask first_task
[2023-10-29T22:18:10.071+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-29T22:18:10.733+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
